src/Symfony/Component/Serializer/Debug/TraceableNormalizer.php

```diff
@@ -35,6 +35,16 @@ public function __construct(
     ) {
     }
 
+    public function getSupportedTypes(?string $format): ?array
+    {
+        // @deprecated remove condition in 7.0
```

Comment:
we should also trigger a deprecation in the constructor for this because currently the deprecation triggered in Serializer will be hidden by this wrapper